tests/testdata/demo-modules/javascript/demo-module-1/can-load.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * Can load function to determine if module already marge in WordPress core.
+ *
+ * @since   n.e.x.t
+ * @package performance-lab
+ */
+
+return function() {
```

Comment:
Looks like this file should be renamed to `can-not-load`

Reply:
You can replace this with: 
```suggestion
return __return_false;
```

Reply:
Sorry, I missed it completely. Thanks for pointing it out. 

> Looks like this file should be renamed to `can-not-load`

According to https://github.com/WordPress/performance/blob/fix/293-mechanism-not-load-module/load.php#L214, if we change the file name, we can't test whether that module is marge or not in the core.